src/main/java/com/server/bugtracker/User/User.java

```diff
@@ -71,12 +71,12 @@ public void setName(String name)
 
     public String getUser_name()
     {
-        return user_name;
+        return username;
     }
 
-    public void setUser_name(String user_name)
+    public void setUsername(String username)
     {
-        this.user_name = user_name;
```

Comment:
Your name change ("username") will not work with the current database set up. Please remove your changes or explain why the change was made. Proper naming conventions for database attributes can be found in the ER diagram located in the repository.